app.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const app = express();
+
+const {getTopics} = require('./controllers/topics.controllers.js');
+
+app.get('/api/topics', getTopics);
+
+app.use((err,request,response,next) => {
+    if (err.status && err.msg)
+      response.status(err.status).send({msg: err.msg});
+    else
+      response.status(400).send({msg: "Bad request"});
+}); 
```

Comment:
These should not be added for this end point as this endpoint does not use any of these error handlers. You can however add an error handler with a generic 500 status code and an appropriate message.